core/spec/models/spree/variant_spec.rb

```diff
@@ -574,14 +574,6 @@
     end
   end
 
-  # Regression test for https://github.com/spree/spree/issues/2744
-  describe "set_position" do
-    it "sets variant position after creation" do
-      variant = create(:variant)
-      expect(variant.position).to_not be_nil
-    end
-  end
-
```

Comment:
Do we really want to remove this test? We still want to persist this behaviour, even if we're removing the callback, right?

Reply:
I wonder if I should write a test for "acts as list" only for Variant, but we feel safer with a spec.
So I added the test! 77aceb0